### DISCUSSION TITLE
Allow Parent Node to Move Independently from Its Children

### DISCUSSION THREAD

**Discussion by Dark-Horizons** [godotengine/godot-proposals#11232] [2D]
In some cases, it's necessary to reposition a parent node independently of its children in the editor. However, when I move the parent node, all its children also move along with it, which means I need to manually adjust each child node afterward.

Is there a shortcut key or an existing feature to make the parent movable without affecting its children in the editor only? If not, wouldn't it be useful to have such a feature in Godot?

**Comment by theraot** [comment]
No, Godot does not have this feature.

What you can do right now is the following: You can reparent the children (drag them out of the parent in the Scene dock), move the parent, and reparent the children back.

This works because when you move nodes in the Scene dock, they keep their global position (if you don't want them to keep their global position when you reparent them you can hold SHIFT while you drop the node in the Scene dock).

**Comment by passivestar** [comment]
Duplicate of https://github.com/godotengine/godot-proposals/issues/9930

**Comment by tetrapod00** [comment]
Closing as duplicate and consolidating into https://github.com/godotengine/godot-proposals/issues/9930. If you have anything to add to the linked proposal issue, feel free to share it in the proposal. You can also leave a 👍 reaction on the proposal
